src/core/sys/posix/sys/types.d

```diff
@@ -992,7 +992,7 @@ else version (Darwin)
 }
 else version (FreeBSD)
 {
-    alias int lwpid_t;
+    alias int   lwpid_t;
```

Comment:
Don't mix functional changes with whitespace changes, if it really matters, raise it as another PR.